### PR TITLE
 Update hardcoded hostname Artifacthub

### LINKED ISSUE #10437: [UI] Registrant: "Artifact Hub" is inappropriately hardcoded
### Current Behavior

See http://localhost:9081/settings?settingsCategory=Registry&tab=Registrants

### Screenshots/Logs
Screenshot 2024-02-27 at 11 13 18 PM

### Environment

- Browser: Safari
- Host OS: Mac 
- Meshery Server Version: stable-v0.7.24

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by yash37158**
**Notes for Reviewers**
This PR addresses the issue where the hostname "Artifacthub" was hardcoded and should be "Artifact Hub". This change ensures consistency and clarity in the UI
This PR fixes #10437 
Screenshot 2024-04-18 at 1 26 41 AM

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MeshModelRegistry/MeshModelComponent.js [hunk 1]
```diff
@@ -222,7 +221,12 @@ const MeshModelComponent_ = ({
       const tempResourcesDetail = [];
 
       for (let registrant of registrants) {
-        let hostname = toLower(registrant?.hostname);
+        let hostname = registrant?.hostname;
+
+        // Check if the hostname is "Artifacthub" and change it to "Artifact Hub"
+        if (hostname === 'Artifacthub') {
```
Comment (leecalcote) [inline ui/components/MeshModelRegistry/MeshModelComponent.js:227]: Thanks. We need to change this at source in the Meshery Integrations spreadsheet.

If the spreadsheet has the correct spelling, then we need to make sure that any processing done in Golang is not changing the name.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/MeshModelRegistry/MeshModelComponent.js (hunk 1 above)
Comment (yash37158) [reply ui/components/MeshModelRegistry/MeshModelComponent.js:227]: Hey @leecalcote, thank you for your review. Could you please share the link to the Meshery Integrations spreadsheet? I'll promptly verify the correct spelling in the spreadsheet and conduct a thorough review of our Golang code to ensure that it doesn't unintentionally alter the names.

Comment (leecalcote) [reply ui/components/MeshModelRegistry/MeshModelComponent.js:227]: Its in the community drive.

Comment (yash37158) [reply ui/components/MeshModelRegistry/MeshModelComponent.js:227]: I've checked the Meshery Integrations spreadsheet and confirmed the correct spelling of the name. However, upon reviewing our Golang code, I couldn't find any instances where the name is being altered unintentionally.

**Comment by vishalvivekm** [comment]
Thank you @yash37158 
Let's discuss this in Meshery dev call today. Add this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#), if you would :)

**Comment by yash37158** [comment]
Yeah sure @vishalvivekm

**Review by Yashsharma1911 — COMMENTED** [review]
@yash37158 thanks for working on this, however this is not issue of name, instead issue of hardcoded link to image that is used in chip, you can take reference in issue, the icon is hardcoded which causing issue of having same icon even if registrant is different then Artifact hub

**Comment by leecalcote** [comment]
Gents, where do we stand on this PR? Still WIP or have we tossed in the towel?

**Comment by leecalcote** [comment]
Gents, where do we stand on this PR? Still WIP or have we tossed in the towel?

**Pull request closed without merging**